### PR TITLE
Html page added

### PR DESCRIPTION
**Pull request by anuragganjiwale**

### CODE REVIEW

File: app/javascript/packs/start.js [hunk 1]
```diff
@@ -56,4 +56,4 @@ function playAnimation(animation){
  html.style.background = "transparent";
  html.style.top = 0;
  html.style.left = 0;
-}
+}
```
Comment (SagareGanesh) [inline app/javascript/packs/start.js]: Is it necessary change?

File: app/javascript/packs/demo_animation.js [hunk 2]
```diff
@@ -12,16 +12,16 @@ show = function(animation){
...
-  html.style.left = 0;
-  html.style.pointerEvents = "none";
+  var anim = bodymovin.loadAnimation(animData);
+  setTimeout(function () {
+    var html = animData.container.getElementsByClassName("lottie")[0];
+    html.style.position = "fixed";
+    html.style.top = 0;
+    html.style.left = 0;
+    html.style.pointerEvents = "none";
 	}, 100);
```
Comment (SagareGanesh) [inline app/javascript/packs/demo_animation.js]: who will be going to indent this code?

File: .gitignore [hunk 3]
```diff
@@ -33,3 +33,4 @@ yarn-debug.log*
 /public/uploads
 /public/users
 /public/animations
+/app/assets/images
```
Comment (SagareGanesh) [inline .gitignore]: which images you are going to add here?

Comment (anuragganjiwale) [reply .gitignore]: The images which i have used for html page

File: .gitignore [hunk 4]
```diff
@@ -32,4 +32,4 @@ yarn-debug.log*
 /coverage
 /public/uploads
 /public/users
-/public/animations
+/public/animations
```
Comment (SagareGanesh) [inline .gitignore]: Is this changes needed?
